app/lib/routes/settings.dart

```diff
@@ -71,6 +78,109 @@ class Settings extends StatelessWidget {
   }
 }
 
+class ShareSettings extends StatelessWidget {
```

Comment:
Can you move this to it's own file?

Maybe you can add a `settings` folder within `lib/widgets` and move this widget to over there

Reply:
Done